src/vision/cfg/hsv-filter-node.cfg

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+PACKAGE = "vision"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+gen.add
+
+gen.add("h_low",    int_t,    	0, "Int defining default Hue low",	5,  0, 360)
+gen.add("s_low",    int_t, 		0, "Int defining default Sat low",    230,  0, 255)
```

Comment:
nit, but it would be nice if these lined up (looks like only part of this line got indented?